describe_version

```diff
@@ -1 +1,69 @@
-./bin/git-fleximod status
+#!/usr/bin/env python
+"""Describes the CESM version and any local modifications"""
+
+from __future__ import print_function
```

Comment:
I don't think that you need this line.

Reply:
Good point - this was a carry-over from the past version of this. I have removed it.